tests/integrationtest/t/executor/simple.test

```diff
@@ -487,3 +487,30 @@ rollback;
 select * from auto_new;
 
 set autocommit = default;
+
+# TestDefaultAuthPluginForCreateUser
+
+connection default;
+set global default_authentication_plugin = 'caching_sha2_password';
```

Comment:
Maybe add something like `set global default_authentication_plugin = 'xxxxx'` to test the validation of the plugin name?

Maybe add statements like `CREATE USER ... IDENTIFIED WITH ...` to test if you can still create users with `mysql_native_plugin` hen the `default_authentication_plugin` is set to `caching_sha2_password`?

Reply:
updated in https://github.com/pingcap/tidb/commit/fa4a9a166f1a4a2c5b43602dab451f4c87d57ada